packages/drivers/iframe-driver/src/outerDocumentServiceFactory.ts

```diff
@@ -69,9 +76,10 @@ export class DocumentServiceFactoryProxy implements IDocumentServiceFactoryProxy
         this.createProxy(frame);
     }
 
-    public async createDocumentService(): Promise<string> {
+    public async createDocumentService(logger?: ITelemetryBaseLogger): Promise<string> {
+        const outerProxyLogger = ChildLogger.create(logger, "OuterProxyIFrameDriver");
         const connectedDocumentService: IDocumentService =
-            await this.documentServiceFactory.createDocumentService(this.resolvedUrl);
+            await this.documentServiceFactory.createDocumentService(this.resolvedUrl, outerProxyLogger);
```

Comment:
This part causes the proxy situation to be a bit different from the live one - the created driver creates a (private) child logger from the provided so it's not exactly the same as the one provided through the proxy